SQL/tgstation_schema.sql

```diff
@@ -591,6 +591,19 @@ CREATE TABLE `discord_links` (
   	`valid` BOOLEAN NOT NULL DEFAULT FALSE,
 	PRIMARY KEY (`id`)
 ) ENGINE=InnoDB;
+---
+---Table structure for table `game_log`
+---
+CREATE TABLE `game_log` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `datetime` datetime NOT NULL,
+  `round_id` int(11) NOT NULL,
+  `ckey` varchar(32) NOT NULL,
+  `loc` varchar(60) NOT NULL,
+  `type` varchar(10) NOT NULL,
+  `message` varchar(1000) NOT NULL,
```

Comment:
You don't appear to have anything preventing a long message from throwing an error.

Reply:
Is there something you would suggest?

Reply:
I don't have a suggestion at this time for this problem. I would investigate if mysql supports a data type like `text` in postgresql

Reply:
Ah yes. Good idea, will change that to use text rather than varchar. mysql supports text and longtext